x-pack/plugins/case/server/common/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './models';
```

Comment:
I am not sure if the `common` folder is included in the webpack bundle. If it is exporting everything will increase the size of it.

Reply:
Christos and I discussed this, currently it's only imported by things in the server so I think we're ok here.